src/DynamoCoreWpf/Windows/BrowserWindow.xaml

```diff
@@ -2,14 +2,20 @@
         xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
         xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
         xmlns:p="clr-namespace:Dynamo.Wpf.Properties"
-        Title="SignInWindow" Height="365" Width="360">
-    <Grid>
+        Title="SignInWindow"
+        Height="365"
+        Width="360">
+    <Grid Background="#222">
+
+        <TextBlock Name="LoadingTextBlock"
+                   FontSize="20"
+                   VerticalAlignment="Center"
+                   HorizontalAlignment="Center"
+                   Foreground="#AAA"
+                   Text="{x:Static p:Resources.BrowserWindowLoading}" />
```

Comment:
Oops, is that a good thing or a bad thing? Does it screw up the rendering? If you have a screenshot it will be great!